lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_GeneratePercentilesFromMeanAndVariance.py

```diff
@@ -130,6 +133,10 @@ def test_simple_data(self):
             percentiles)
         self.assertArrayAlmostEqual(result.data, result_data)
 
+    @ManageWarnings(
+        ignored_messages=["invalid value encountered",
```

Comment:
What's this `invalid value encountered` warning about?

Reply:
/opt/scitools/environments/default/2018_01_19/lib/python2.7/site-packages/scipy/stats/_distn_infrastructure.py:1904: RuntimeWarning: invalid value encountered in multiply
  upper_bound = self.b * scale + loc

Reply:
I'm slightly concerned about this, do you think we should look into this a bit before we suppress it?

Reply:
I would rather open a new ticket to investigate. It is a really obvious error that anyone writing the unit tests would have seen (it repeats about 5 or 6 times) so I have to assume that they have seen and discounted it.

Reply:
+1 for an investigation ticket on this.

Reply:
I've just had a chat to @gavinevans , who wrote this code. This warning happens because the scipy routine is coping with a NaN in the input somewhere, or creating a NaN. If you read the docstring for this unit test it describes what is happening. Basically Gavin put something in the plugin to deal with any NaNs but didn't suppress the warning so it is still being raised. I'm happy to suppress the warning in this PR, and we should add suppressing this warning to the list of warnings we probably need to think about suppressing in the plugins. (Suppressing warnings in plugins is out of scope of this PR.)